Makefile

```diff
@@ -171,3 +171,15 @@ clean:
 	./deployments/image/configmap/${BINARY} \
 	./deployments/image/configmap/${CONF_FILE} \
 	./deployments/image/admission/${POD_ADMISSION_CONTROLLER_BINARY}
+
+# Run the e2e tests, this assumes yunikorn is running under yunikorn-ns namespace
+.PHONY: e2e_test
```

Comment:
It is not clear to me why we need 2 phases here, `e2e_test` and `ci_test`. I think they are both e2e tests. If they belong to different buckets, can we support that by providing options? Such as `make e2e_test TEST_SUITS=integration,ci`.. may not be a good example, just to propose a way to configure this : )